sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -139,6 +139,25 @@ def done(self):
       self._state = self.DONE_STATE
 
 
+class _PartitionBuffer(object):
```

Comment:
Is there any reason this is a class that has a single method rather than just a function?

Reply:
This class is removed now.